script/Ciel/Day 8/08_01_CIEL08_5.txt

```diff
@@ -573,59 +677,68 @@ The sister tosses away her blood-soaked halberd with a dejected look on her face
 [sha:b7f4c2c71521290760560e00ee52a3a703ccf7b0]{
 -- Page 466, Offset 19432.
 -- 死んだ魚のような目で、肺の中の空気をはき出す。
-With lifeless eyes, she exhales the air contained in her lungs.
+With lifeless eyes, she lets out a long, slow sigh.
 }
 [sha:2c2125ff4e23541221adbf8ddfb21f8e3e214eee]{
 -- Page 466, Offset 19433.
 -- それは魂が抜け落ちたかのような、あまりにも憂鬱な溜息だった。
-// 魂が抜ける is also an idiom, but I chose to translate it literally here
-It was a sigh so remarkably melancholic, it could have carried away her soul.
+It was an incredibly melancholy sigh,
```

Comment:
melancholic

Reply:
Melancholy sigh seems to be some kind of weird collocation, so why not.

I'll bet you something that it'll appear a lot in the public feedback later, though!

Reply:
I will delicately place their feedback in the 'spricht kein English' pile where it belongs